package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "jupyter",
     "displayName": "Jupyter",
-    "version": "2022.6.110",
+    "version": "2022.6.120",
```

Comment:
I believe we need the same kind of change in the package-lock.json. There should be two instances there. Please, let the know if I'm wrong 🙇‍♂️

Reply:
I did a regen at first, but it tweaked a bit more of the package lock so I just went with manual...